src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -424,6 +462,31 @@ export const StartDate = ({
   )
 }
 
+type TooSoonSwitcherErrorModalProps = {
```

Comment:
Nit: this file is already huge, maybe we can split error modal and related utils to separate module?

Reply:
Sure, that's non unreasonable 🙃